### ISSUE TITLE
Make an index page for the Adam server 

### ISSUE THREAD

**Issue by mydatascience** [lf-edge/adam#25]
When I go to the page I'd like to see at least something like:

> Hello. Welcome to ADAM. Here are the Endpoints: 
> 
> The following are the admin endpoints:
> 
>     GET /onboard - list all onboard certificates
>     GET /onboard/{cn} - get a specific onboard certificate
>     POST /onboard - upload a new onboarding certificate
>     DELETE /onboard - clear all onboarding certificates
>     DELETE /onboard/{cn} - delete a specific onboarding certificate
>     GET /device - list all devices
>     GET /device/{uuid} - get details of one device
>     GET /device/{uuid}/config - get config for one device
>     PUT /device/{uuid}/config - update config for one device
>     GET /device/{uuid}/logs - get all known logs for one device; set header X-Stream=true to stream all new logs instead
>     GET /device/{uuid}/info - get all known info messages for one device; set header X-Stream=true to stream all new info instead
>     POST /device - create a new device
>     DELETE /device - delete all devices
>     DELETE /device/{uuid} - delete one specific device

Even better if I see if I have some EVE connected to ADAM

**Comment by rvs** [comment]
Yeah, that's been on @deitch and my TODO list for some time (and I suppose it is also related to #28 since you can serve that one up as well)

That said, we still not quite out of the weeds on basic functionality -- patches are VERY welcome, of course.
